TUTORIAL.md

```diff
@@ -17,7 +17,7 @@ contracts.ruby brings code contracts to the Ruby language. Code contracts allow
 A simple example:
 
 ```ruby
-Contract Contracts::Num, Contracts::Num => Contracts::Num
+Contract Contracts::Builtin::Num, Contracts::Builtin::Num => Contracts::Builtin::Num
```

Comment:
I think, we don't want to expose this information to users. We want them to still either:
- include `Contracts::Core` and use `Contracts::Maybe` (or `C::Maybe`)
- include `Contracts::Core`, `Contracts::Builtin` and use `Maybe`

So in `TUTORIAL.md` good changes are only for error messages and `validator` list